cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -18,6 +18,7 @@ module Cardano.DbSync
   , GenesisHash (..)
   , NetworkName (..)
   , SocketPath (..)
+  , DB.MigrationDir (..)
```

Comment:
Why re-export that here instead of just importing it from `Cardano.Db` ?

Reply:
I can do that as well.

Reply:
Fixed.

Reply:
Wait. I now remember why. Because then I depend on `Cardano.Db` as well as `Cardano.DbSync`.